wings/events.py

```diff
@@ -167,6 +167,20 @@ class OrderType(Enum):
     LIMIT = 2
 
 
+class FeeType(Enum):
+    ADD_BASE = 1
+    SUB_BASE = 2
+    ADD_QUOTE = 3
+    SUB_QUOTE = 4
+    ADD_OTHER = 5
+    SUB_OTHER = 6
+
+
+class TradeFee(NamedTuple):
+    type: FeeType
+    amount: float
+    other_symbol: str = None
+
```

Comment:
These are not events and belong in a separate file.

Reply:
agreed, but lets do this in another PR because OrderType/TradeType should be moved too.

Reply:
ok fine